diff -ruN fedora-release-12-orig/fedora.repo fedora-release-12/fedora.repo
--- fedora-release-12-orig/fedora.repo	2009-11-02 17:15:04.000000000 -0500
+++ fedora-release-12/fedora.repo	2024-07-30 10:41:13.000000000 -0400
@@ -1,29 +1,33 @@
 [fedora]
 name=Fedora $releasever - $basearch
 failovermethod=priority
-#baseurl=http://download.fedoraproject.org/pub/fedora/linux/releases/$releasever/Everything/$basearch/os/
-mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=fedora-$releasever&arch=$basearch
+baseurl=http://dl.fedoraproject.org/pub/archive/fedora/linux/releases/$releasever/Everything/$basearch/os/
 enabled=1
 metadata_expire=7d
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$basearch
+# Exclude linear algebra libs that should be installed by the Cell SDK instead
+exclude=lapack lapack-devel blas blas-devel
 
 [fedora-debuginfo]
 name=Fedora $releasever - $basearch - Debug
 failovermethod=priority
-#baseurl=http://download.fedoraproject.org/pub/fedora/linux/releases/$releasever/Everything/$basearch/debug/
-mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=fedora-debug-$releasever&arch=$basearch
+baseurl=http://dl.fedoraproject.org/pub/archive/fedora/linux/releases/$releasever/Everything/$basearch/debug/
 enabled=0
 metadata_expire=7d
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$basearch
+# Exclude linear algebra libs that should be installed by the Cell SDK instead
+exclude=lapack lapack-devel blas blas-devel
 
 [fedora-source]
 name=Fedora $releasever - Source
 failovermethod=priority
-#baseurl=http://download.fedoraproject.org/pub/fedora/linux/releases/$releasever/Everything/source/SRPMS/
-mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=fedora-source-$releasever&arch=$basearch
+baseurl=http://dl.fedoraproject.org/pub/archive/fedora/linux/releases/$releasever/Everything/source/SRPMS/
 enabled=0
 metadata_expire=7d
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$basearch
+# Exclude linear algebra libs that should be installed by the Cell SDK instead
+exclude=lapack lapack-devel blas blas-devel
+
diff -ruN fedora-release-12-orig/fedora-updates.repo fedora-release-12/fedora-updates.repo
--- fedora-release-12-orig/fedora-updates.repo	2009-11-02 17:15:04.000000000 -0500
+++ fedora-release-12/fedora-updates.repo	2024-07-30 10:40:11.000000000 -0400
@@ -1,26 +1,30 @@
 [updates]
 name=Fedora $releasever - $basearch - Updates
 failovermethod=priority
-#baseurl=http://download.fedoraproject.org/pub/fedora/linux/updates/$releasever/$basearch/
-mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=updates-released-f$releasever&arch=$basearch
+baseurl=http://dl.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/$basearch/
 enabled=1
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$basearch
+# Exclude linear algebra libs that should be installed by the Cell SDK instead
+exclude=lapack lapack-devel blas blas-devel
 
 [updates-debuginfo]
 name=Fedora $releasever - $basearch - Updates - Debug
 failovermethod=priority
-#baseurl=http://download.fedoraproject.org/pub/fedora/linux/updates/$releasever/$basearch/debug/
-mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=updates-released-debug-f$releasever&arch=$basearch
+baseurl=http://dl.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/$basearch/debug/
 enabled=0
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$basearch
+# Exclude linear algebra libs that should be installed by the Cell SDK instead
+exclude=lapack lapack-devel blas blas-devel
 
 [updates-source]
 name=Fedora $releasever - Updates Source
 failovermethod=priority
-#baseurl=http://download.fedoraproject.org/pub/fedora/linux/updates/$releasever/SRPMS/
-mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=updates-released-source-f$releasever&arch=$basearch
+baseurl=http://dl.fedoraproject.org/pub/archive/fedora/linux/updates/$releasever/SRPMS/
 enabled=0
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$basearch
+# Exclude linear algebra libs that should be installed by the Cell SDK instead
+exclude=lapack lapack-devel blas blas-devel
+
diff -ruN fedora-release-12-orig/fedora-updates-testing.repo fedora-release-12/fedora-updates-testing.repo
--- fedora-release-12-orig/fedora-updates-testing.repo	2009-11-11 11:21:04.000000000 -0500
+++ fedora-release-12/fedora-updates-testing.repo	2024-07-30 10:39:09.000000000 -0400
@@ -1,26 +1,30 @@
 [updates-testing]
 name=Fedora $releasever - $basearch - Test Updates
 failovermethod=priority
-#baseurl=http://download.fedoraproject.org/pub/fedora/linux/updates/testing/$releasever/$basearch/
-mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-f$releasever&arch=$basearch
-enabled=0
+baseurl=http://dl.fedoraproject.org/pub/archive/fedora/linux/updates/testing/$releasever/$basearch/
+enabled=1
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$basearch
+# Exclude linear algebra libs that should be installed by the Cell SDK instead
+exclude=lapack lapack-devel blas blas-devel
 
 [updates-testing-debuginfo]
 name=Fedora $releasever - $basearch - Test Updates Debug
 failovermethod=priority
-#baseurl=http://download.fedoraproject.org/pub/fedora/linux/updates/testing/$releasever/$basearch/debug/
-mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-debug-f$releasever&arch=$basearch
+baseurl=http://dl.fedoraproject.org/pub/archive/fedora/linux/updates/testing/$releasever/$basearch/debug/
 enabled=0
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$basearch
+# Exclude linear algebra libs that should be installed by the Cell SDK instead
+exclude=lapack lapack-devel blas blas-devel
 
 [updates-testing-source]
 name=Fedora $releasever - Test Updates Source
 failovermethod=priority
-#baseurl=http://download.fedoraproject.org/pub/fedora/linux/updates/testing/$releasever/SRPMS/
-mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=updates-testing-source-f$releasever&arch=$basearch
+baseurl=http://dl.fedoraproject.org/pub/archive/fedora/linux/updates/testing/$releasever/SRPMS/
 enabled=0
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$basearch
+# Exclude linear algebra libs that should be installed by the Cell SDK instead
+exclude=lapack lapack-devel blas blas-devel
+
